src/scss/components/optional/_ratings.scss

```diff
@@ -21,12 +21,8 @@
     $star-size--medium          : 28px;
     $star-size--large           : 42px;
 
-  // https://github.com/justeat/f-icons/blob/master/icons/star-filled.svg
-  @warn 'To use the rating component you will need to add references to the SVGs in f-icons';
```

Comment:
Just a note – the consuming application will still need to add the silent extend mixin definition for this to work (like Menu does), so it may be worth updating the comment for that?